test/sanity/validate-modules/main.py

```diff
@@ -1226,16 +1226,17 @@ def _validate_argument_spec(self, docs, spec, kwargs):
                 self.reporter.error(
                     path=self.object_path,
                     code=327,
-                    msg=('Default value from the documentation (%r) is not compatible '
-                         'with type %r defined in the argument_spec' % (doc_options_arg.get('default'), _type))
+                    msg=("Argument '%s' in documentation defines default as '%s' "
```

Comment:
Please switch back to just using `%r` here instead of `'%s'` as that assists in providing more information about the original type.

Reply:
Fixed.